frodo.js

```diff
@@ -1,73 +1,76 @@
-const firstElement = array[0];
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.